Marlin/ultralcd.cpp

```diff
@@ -1110,13 +1110,24 @@ static void lcd_control_volumetric_menu() {
 #ifdef HAS_LCD_CONTRAST
   static void lcd_set_contrast() {
     if (encoderPosition != 0) {
-      lcd_contrast -= encoderPosition;
-      lcd_contrast &= 0x3F;
+      #ifdef U8GLIB_LM6059_AF
+        lcd_contrast += encoderPosition;
```

Comment:
Roxy, I did not mean at all to sound mean or angry at any of you! I meant I
am so glad the environment is no longer toxic here! I appreciate all your
hard work and look forward to continue collaborating with the team!

On Wed, 1 Jun 2016 at 20:14, Roxy-3DPrintBoard <notifications@github.com>
wrote:

> In Marlin/ultralcd.cpp
> <https://github.com/MarlinFirmware/Marlin/pull/2105#discussion_r65348309>:
>
> > @@ -1110,13 +1110,24 @@ static void lcd_control_volumetric_menu() {
> >  #ifdef HAS_LCD_CONTRAST
> >    static void lcd_set_contrast() {
> >      if (encoderPosition != 0) {
> > -      lcd_contrast -= encoderPosition;
> > -      lcd_contrast &= 0x3F;
> > +      #ifdef U8GLIB_LM6059_AF
> > +        lcd_contrast += encoderPosition;
>
> @emartinez167 <https://github.com/emartinez167> I fully understand your
> statements. We would like you to give us a second chance. If you even think
> you were treated less politely than you should have been, please send an @
> message to ThinkyHead or me. My guess is we won't be getting any messages
> from you going forward. We are very serious about keeping this a healthy
> environment for both users and developers.
>
> —
> You are receiving this because you were mentioned.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/MarlinFirmware/Marlin/pull/2105/files/082ed3beedfaffaa7fb8b8e2ae4fa2f0f9f8aca4#r65348309>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AGcPb2IDnwCS4FHmvu3x1s4yCihqoqkvks5qHXejgaJpZM4Ed7jG>
> .
>